src/pages/players/[id]-[username].astro

```diff
@@ -56,7 +55,7 @@ const leaderboardEntries = player?.leaderboard_entries
       </div>
       <Widget title="Recent Matches" label="Closed Beta Ranked">
         <div>
-          {playerMatches.matches.map((match) => <MatchPreview match={match} mainPlayerId={playerId} />)}
+          {playerMatches.matches.map((match) => <MatchPreview match={match} mainPlayerId={playerId} client:idle />)}
```

Comment:
I am not 100% sure that it is the best tooltip fix. I am not yet very familiar with those `client:`. Feel free to suggest any better solution.

Reply:
Yes this is correct!

Reply:
Do you have a good explanation why this is a fix? I am not sure I totally get it